addons/controllers/clusterbootstrap_controller.go

```diff
@@ -1100,65 +1100,74 @@ func (r *ClusterBootstrapReconciler) GetDataValueSecretNameFromBootstrapPackage(
 		err            error
 	)
 
-	if cbPkg.ValuesFrom == nil || cbPkg.ValuesFrom.Inline != nil {
-		packageRefName, _, err = util.GetPackageMetadata(r.context, r.aggregatedAPIResourcesClient, cbPkg.RefName, cluster.Namespace)
-		if packageRefName == "" || err != nil {
-			// Package.Spec.RefName and Package.Spec.Version are required fields for Package CR. We do not expect them to be
-			// empty and error should not happen when fetching them from a Package CR.
-			r.Log.Error(err, fmt.Sprintf("unable to fetch Package.Spec.RefName or Package.Spec.Version from Package %s/%s",
-				cluster.Namespace, cbPkg.RefName))
-			return "", err
-		}
+	packageRefName, _, err = util.GetPackageMetadata(r.context, r.aggregatedAPIResourcesClient, cbPkg.RefName, cluster.Namespace)
+	if packageRefName == "" || err != nil {
```

Comment:
@vijaykatam Do we always want to do package metadata existence check? Cause if not found, this would return from this function